clients/go/gengo.go

```diff
@@ -600,7 +600,7 @@ func parseResponseCode(s *spec.Swagger, op *spec.Operation, capOpID string) stri
 	errorType, _ := swagger.OutputType(s, op, 500)
 	buf.WriteString(fmt.Sprintf(`
 	default:
-		return %s&%s{Message: "Unknown response"}
+		return %s&%s{Message: fmt.Sprintf("Unknown status code %%%%%%%%v", resp.StatusCode)}
```

Comment:
lol what's the conversion happening here that requires `%%%%%%%%v` -> `%v`?

Reply:
this string gets formatted a bunch of times so I needed to escape the `%v` as many times as it's being formatted

Reply:
I almost gave up and just used `+`